public/Lab-Files/Lesson03/Lesson04/Assignment/Team-Project/index.html

```diff
@@ -0,0 +1,75 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Blue Team - Home</title>
+    <link rel="stylesheet" href="css/styles.css" type="text/css" />
+</head>
+<body>
+    <h2 class="top-head">Recent From Blog</h2>
+    <p class="top-head">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ducimus, sit, assumenda natus officia nihil est nesciunt obcaecati doloremque ab provident iusto adipisci quisquam vel perferendis saepe esse numquam? Nemo, eligendi!</p>
+        <div class="main">
+            <img src="img/keyboard.JPG" class="keyboard"></img>
+            <h4>Terrible Typing Form</h4>
+            <p class="laptop-text">
+            Lorem ipsum dolor sit amet, consectetuer adipiscing elit,
+            sed diam nonummy nibh euismod tincidunt ut laoreet dolore
+            magna aliquam erat volutpat.  </p>
+         <a  href="https://www.austincodingacademy.com/" class="link2" target="blank">Read more</a>
+            
+        </div>
+        
+        <div class="top">
+            <img src="img/laptop.jpg" class="laptop"></img>
+            <h5>Helo</h5>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. </p>
+        </div>
+        
+        <div class="mid">
+            <img src="img/desk.jpg" class="desk"></img>
+            <h5>hey</h5>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. </p>
+        </div>
+        
+        <div class="bottom">
+            <img src="img/window.jpg" class="window"></img>
+             <h5>Helo</h5>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. </p>
+            
+        </div>
+ 
+  <header>
+    <!-- header content goes in here -->
+  </header>
+
+  <footer>
+    <!-- footer content goes in here -->
+    <div class="footer">
+        <hr>
+        <h2 class=h2_1>Why Profession?</h2>
+        <ul class=list_1>
+            <li><img src="img/green_checkmark_small.png" alt="My Image">Clean and Corporate Webdesign</li>
+            <li>Well Coded HTML template</li>
+            <li>Multiple Layout options</li>
+        </ul>    
+        <hr class=vert2>
+        <h2 class=h2_2>Main Features</h2>
+        <img src="img/green_checkmark_small.png" alt="checkmark4"class=check4><img>
+        <h5>Bootstrap 3.x & SASS</h5>
+        <img src="img/green_checkmark_small.png" alt="checkmark6"class=check6><img>
+        <h5>Super lighting page speed</h5>
+        <img src="img/green_checkmark_small.png" alt="checkmark5"class=check5><img>
+        <h5>Well documented code</h5>
+        <hr class=vert3>
+        <h2 class=h2_3>Transparent Pricing</h2>
+        <h5>Are you interested in our Services? Dont hesitate and contact us.</h5>
+        <a href="default.asp" target="_blank" class=link2>Check Pricing</a>
+        <hr>
+    </div>
+    
+  </footer>
+    </div>
+     
+     <script type="text/javascript" src="js/script.js"></script>
+</body>
+</html>
```

Comment:
Make sure you validate your code, there are a lot of errors.  For example in lines 12, 23, 29, and 35 you did not add the alt attribute to your images. Also, it does not appear like you used your header, so it would be best to remove that element. 

Overall, I see major improvement on these assignments than the previous lesson, keep working hard! Don't forget to validate your code, and now that it seems you are understanding more of the concepts, try to focus on organizing your code.